CONTRIBUTING.md

```diff
@@ -68,8 +68,22 @@ accept your pull requests.
 
         npm run fix
 
+## Running the tests in GCP services
```

Comment:
You're going to get bitten by the fact that this file is templatized too, again you can just add to the `excludes` in `synth.py`.

Reply:
Noted, thank you

Reply:
I added it [here](https://github.com/googleapis/nodejs-logging/blob/496c58287c13e24a73ddea59829378e1ce78dcbf/synth.py#L38)